### PR TITLE
Remove --etcd_servers flag from kubelet and proxy.

### PR DESCRIPTION
**Pull request by erictune**
All the distros that use this have been updated,
or have PRs out to update them, or owners
have been asked to fix RPMs.

Removing this prevents further use of this model.

Remove now dead code: EtcdClientOrDie

Remove now dead pkg/proxy/config/etcd.go.

Remove unused imports.

### CODE REVIEW

**Comment by vmarmol** [comment]
LGTM

**Comment by dchen1107** [comment]
LGTM Thanks!

**Comment by timothysc** [comment]
@erictune for clarity, we need to update everything to use the --api_server for kubelet and remove all old args?

**Comment by erictune** [comment]
@timothysc 
All the scripts in this repo that use `--etcd_servers` have been updated, or have PRs out to update them.
For all known (to me) uses in other repos, I have notified the owners to fix their RPMs.

**Comment by alex-mohr** [comment]
Kicked shippable build again -- timeout.

**Comment by derekwaynecarr** [comment]
Always happy to see code I previously debugged getting out of the code base ;-)

**Comment by a-robinson** [comment]
So much LGTM

File: cmd/kube-proxy/app/server.go [hunk 1]
```diff
@@ -59,8 +56,6 @@ func NewProxyServer() *ProxyServer {
 
 // AddFlags adds flags for a specific ProxyServer to the specified FlagSet
 func (s *ProxyServer) AddFlags(fs *pflag.FlagSet) {
-	fs.StringVar(&s.EtcdConfigFile, "etcd_config", s.EtcdConfigFile, "The config file for the etcd client. Mutually exclusive with -etcd_servers")
```
Comment (thockin) [inline cmd/kube-proxy/app/server.go]: In hindsight we should have left the flags as no-ops with a warning.  This is an annoyance in that old configs no longer work correctly.

Comment (timothysc) [inline cmd/kube-proxy/app/server.go]: @thockin + @roberthbailey perhaps calling out these changes in the changelog for the future.  So folks will be aware of sea-changes that will break their updates.  

It's how we roll errata's. 
@eparis ^ FYI.

**Pull request merged**